packages/m/mitojs.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "@zyf2e/mitojs",
```

```suggestion
  "name": "mitojs",
```
This should match the filename, we only allow alphanum, periods, underscore & hyphens in package names